src/test/java/seedu/address/logic/parser/profile/ProfileCommandParserTest.java

```diff
@@ -35,4 +54,55 @@ public void parse_profileOption_failure() {
                 ProfileCommand.OPTION_UNKNOWN + ProfileCommand.VALID_FLAGS);
     }
 
+    @Test
+    public void parse_profileOption_add() {
+        Profile profile = new ProfileBuilder().build();
+        try {
+            AddProfileCommand command =
+                    (AddProfileCommand) parser.parse(ProfileUtil.getAddProfileCommand(profile));
+            assertEquals(new AddProfileCommand(profile), command);
+        } catch (ParseException e) {
+            fail();
+        }
+    }
+
+    @Test
+    public void parse_profileOption_delete() {
+        try {
+            DeleteProfileCommand command = (DeleteProfileCommand) parser.parse(
+                    " " + PREFIX_OPTION + DeleteProfileCommand.COMMAND_OPTION + " "
+                            + INDEX_FIRST_PROFILE.getOneBased());
+            assertEquals(new DeleteProfileCommand(INDEX_FIRST_PROFILE), command);
+        } catch (ParseException e) {
+            fail();
+        }
+    }
+
+    @Test
+    public void parseCommand_edit() {
```

Comment:
Minor inconsistency in the naming of test cases, that said, maybe it would be better to name the test methods in the format `featureUnderTest_testScenario_expectedBehavior()`, following the [Java conventions](https://se-education.org/guides/conventions/java/intermediate.html), in this case, my suggestion is to name it `parse_editOption()`.

Reply:
Okay thanks for the suggestion! I will include the expectedBehavior as well, to keep it consistent with the above 3 methods where they mentioned the expectedBehavior.